frontend/src/metabase-types/api/database.ts

```diff
@@ -50,6 +50,10 @@ export interface Database extends DatabaseData {
   tables?: Table[];
 }
 
+export type DatabaseWithTables = Database & {
```

Comment:
Several state updates can cause tables to be removed. I'd rather not have a type like that and ask people to use just `Database` and check for `tables` being undefined

Reply:
Good point, done!